src/popup.jsx

```diff
@@ -24,6 +25,10 @@ function Popup() {
     })
   }, [])
 
+  function handleChange(event) {
```

Comment:
I would change the name to something more specific to make the code more clear. Something like `handleKeyTypeSelect`